pkg/controller/config/v1alpha1/validation/config.go

```diff
@@ -11,17 +11,17 @@ func ValidateExtensionConfig(config *configv1alpha1.ExtensionConfig) field.Error
 	allErrs := field.ErrorList{}
 	var rootPath *field.Path
 
-	validDaemonNames := sets.New(configv1alpha1.SystemdTimesyncd, configv1alpha1.NTPD)
+	validDaemonNames := sets.New(configv1alpha1.SystemdTimesyncd, configv1alpha1.NTPD, configv1alpha1.None)
 
 	if config.NTP != nil {
 		// Make sure daemon name is valid
 		if !validDaemonNames.Has(config.NTP.Daemon) {
 			allErrs = append(allErrs, field.NotSupported(rootPath.Child("daemon"), config.NTP.Daemon, validDaemonNames.UnsortedList()))
 		}
 
-		// Check if user configured systemd-timesyncd daemon with ntpd config
-		if config.NTP.Daemon == configv1alpha1.SystemdTimesyncd && config.NTP.NTPD != nil {
-			allErrs = append(allErrs, field.Forbidden(rootPath.Child("ntpd"), "NTP daemon not allowed in systemd config"))
+		// Check if user configured systemd-timesyncd OR None daemon with ntpd config
+		if (config.NTP.Daemon == configv1alpha1.SystemdTimesyncd || config.NTP.Daemon == configv1alpha1.None) && config.NTP.NTPD != nil {
```

Comment:
To check if it is not `configv1alpha1.NTPD` but `config.NTP.NTPD` makes the if shorter and no need to adopt it in case there are additional daemons in the future.
```suggestion
		if config.NTP.Daemon != configv1alpha1.NTPD && config.NTP.NTPD != nil {
```